src/components/Todo/Todo.module.css

```diff
@@ -0,0 +1,5 @@
+.title {
+    font-family: 'Courier New', Courier, monospace;
+    text-align: center;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6103729)